csharp/source/autowrap/csharp/csharp.d

```diff
@@ -0,0 +1,1061 @@
+module autowrap.csharp.csharp;
+
+import autowrap.csharp.boilerplate;
+import autowrap.csharp.common;
+import autowrap.reflection;
+
+import std.ascii;
```

Comment:
Please don't use unqualified global imports. There aren't any in the rest of the codebase.